### ISSUE TITLE
[Accessibility]: Code Insights - focus is not obvious after clicking Done button

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41664]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33494

### Problem description

Navigate to create a new search insights `/insights/create/search`

After activating the "Done" button the data series panel collapses, and all focus is no longer visible.

See loom below.

### Expected behavior

Something should be highlighted to let the user know where they are on the page.

The series panel highlights when hovered, maybe it makes sense to focus there.

Example of panel focus:

![PixelSnap 2022-09-14 at 12 51 22](https://user-images.githubusercontent.com/1855233/190226894-6e8a5188-e261-43d1-8044-837876e3dbfd.png)

### Additional details

https://www.loom.com/share/96c3046c94b94ed9a0bac8ca0db9c4c2

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
